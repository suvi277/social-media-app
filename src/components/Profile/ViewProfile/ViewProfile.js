import React from 'react';
import './ViewProfile.scss';

const ViewProfile = ({ firstName, lastName, email, children }) => {
	return (
		<div className="profile-info">
			<h5>{`${firstName} ${lastName}`}</h5>
			<p>{email}</p>
			{children}
		</div>
	);
};

export { ViewProfile };
