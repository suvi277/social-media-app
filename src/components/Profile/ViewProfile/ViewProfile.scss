.profile-info {
    padding: 20px;
    border: 1px solid #ccc;
    border-radius: 5px;
    width: 100%;
}