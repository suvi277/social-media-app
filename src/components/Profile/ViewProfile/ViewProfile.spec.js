import React from 'react';
import { shallow } from 'enzyme';
import { ViewProfile } from './ViewProfile';

describe('ViewProfile Component', () => {
	const wrapper = shallow(<ViewProfile />);

	it('renders as expected', () => {
		expect(wrapper).toMatchSnapshot();
	});
	//To Complet Test Cases
});
