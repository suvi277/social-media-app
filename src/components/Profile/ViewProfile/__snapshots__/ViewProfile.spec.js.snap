// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ViewProfile Component renders as expected 1`] = `ShallowWrapper {}`;
